target_python_sdk/cookies.py

```diff
@@ -99,5 +99,5 @@ def create_target_cookie(cookies):
     return {
         'name': TARGET_COOKIE,
         'value': "|".join(serialized_cookies),
-        'maxAge': math.ceil(max_age / MILLISECONDS_IN_SECOND)
+        'maxAge': math.ceil(max_age // MILLISECONDS_IN_SECOND)
```

Comment:
`//` is an integer division which is really `floor(max_age / MILLISECONDS_IN_SECOND)`.  Having `ceil` doesn't seem to be needed.

Is `maxAge` expected to be an integer?  `//` can return return float, but if `max_age` is always an integer, then it shouldn't be a problem.

```
>>> 3 // 2
1
>>> 3.0 // 2
1.0
```

Reply:
@5amfung maxAge "should" be an int, but that's not enforced.  The intent of this change was to remain back compat with python 2.7, but I'm realizing now these results would differ from the nodejs impl since this would round down instead of up.  In order to work with all python versions and stay consistent with other sdks I'm updating the statement to:
`'maxAge': math.ceil(float(max_age) / MILLISECONDS_IN_SECOND)`